.gitignore

```diff
@@ -1,3 +1,7 @@
-_build
-_coverage
+_*
 owi-out
+*~
+headers-cc.txt
+headers-ml.txt
+.header-check-more
+check-headers.undo
```

Comment:
Which files were ignored but generated by our workflow? If they are specific to your own tooling, wouldn't they be better suited to be in .git/info/exclude?

Reply:
What do you mean by **our workflow** ? Isn't **my workflow** part of **our workflow** ?

Also, do you really care about any **other** contributor adding a few lines in `.gitignore` ? In general, it's a small concession, but having more contributors to an open-source tool is usually worth the trouble...

Reply:
> What do you mean by our workflow ? Isn't my workflow part of our workflow ?

Taking "our workflow' to mean the one implemented in this repository, I don't think yours is part of ours, even after merging this PR.

> Also, do you really care about any other contributor adding a few lines in .gitignore

Yes, most projects I know of try to keep their .gitignore light, I believe it is good practice to do the same.

That being said, I don't think this discussion will turn into anything more than bikeshedding, so let's just say that without more justification for these additions, I am against, and see what others think.